www_src/pages/discover/mock.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "id": 1,
+        "version": "0.0.0",
```

Comment:
do semvers make sense for what is essentially simple version control? This feels like it should just be a plain revision number.